dbos/_sys_db.py

```diff
@@ -367,7 +367,7 @@ def set_workflow_status(
         with self.engine.begin() as c:
             stmt = (
                 sa.update(SystemSchema.workflow_status)
-                .where(SystemSchema.workflow_inputs.c.workflow_uuid == workflow_uuid)
+                .where(SystemSchema.workflow_status.c.workflow_uuid == workflow_uuid)
```

Comment:
Saw a warning pop up about this line and fixed it. The reason we never noticed it before is because this entire function was never used until we added CANCEL recently.